metersink/api.py

```diff
@@ -1,15 +1,15 @@
+"""describes the api endpoint of the metering tool"""
 import argparse
 import logging
 from pprint import pformat
-
 from flask import Flask, request, json
 
-from metersink.lib import *
+from metersink.lib import get_config, get_sinks, output_file
+from metersink.output_odoo import odoo_handle
 
 app = Flask(__name__)
```

Comment:
Is it necessary to use a global variable here?

Reply:
it is necessary to call the app decorator. I am not aware of other possibilities to provide it